src/hivedscheduler/config/hivedscheduler.yaml

```diff
@@ -18,5 +18,5 @@
 
 service_type: "k8s"
 
-webservice-port: 30096
+webservice-port: 10096
```

Comment:
do not need to change, scheduler always runs on master node

Reply:
Sure, it only runs on master node. But limit it to 1024-19999 for consistency.